src/Java.Interop/Java.Interop/JavaPeerableExtensions.cs

```diff
@@ -11,5 +12,40 @@ public static class JavaPeerableExtensions {
 			JniPeerMembers.AssertSelf (self);
 			return JniEnvironment.Types.GetJniTypeNameFromInstance (self.PeerReference);
 		}
+
+		public static bool TryJavaCast<
```

Comment:
We should probably add XML documentation to these public methods explaining what they do for IntelliSense.

Reply:
Done.